---
title: Common and Rare Variant Prediction and Penetrance of IBD in a Large, Multi-ethnic,
  Health System-based Biobank Cohort
authors:
- Kyle Gettler
- Rachel Levantovsky
- Arden Moscati
- Mamta Giri
- Yiming Wu
- Nai-Yun Hsu
- Ling-Shiang Chuang
- Aleksejs Sazonovs
- Suresh Venkateswaran
- Ujunwa Korie
- Colleen Chasteau
- National Institute of Diabetes, Digestive UK IBD Genetics Consortium
- Kidney Diseases Inflammatory Bowel Disease Genetics Consortium
- Richard H. Duerr
- Mark S. Silverberg
- Scott B. Snapper
- Mark J. Daly
- Dermot P. McGovern
- Steven R. Brant
- John D. Rioux
- Subra Kugathasan
- Carl A. Anderson
- Yuval Itan
- Judy H. Cho
date: '2021-04-01'
publishDate: '2024-06-26T22:03:36.482694Z'
publication_types:
- article-journal
publication: '*Gastroenterology*'
doi: 10.1053/j.gastro.2020.12.034
abstract: 'BACKGROUND AND AIMS: Polygenic risk scores (PRS) may soon be used to predict
  inflammatory bowel disease (IBD) risk in prevention efforts. We leveraged exome-sequence
  and single nucleotide polymorphism (SNP) array data from 29,358 individuals in the
  multiethnic, randomly ascertained health system-based BioMe biobank to define effects
  of common and rare IBD variants on disease prediction and pathophysiology. METHODS:
  PRS were calculated from European, African American, and Ashkenazi Jewish (AJ) reference
  case-control studies, and a meta-GWAS run using all three association datasets.
  PRS were then combined using regression to assess which combination of scores best
  predicted IBD status in European, AJ, Hispanic, and African American cohorts in
  BioMe. Additionally, rare variants were assessed in genes associated with very early-onset
  IBD (VEO-IBD), by estimating genetic penetrance in each BioMe population. RESULTS:
  Combining risk scores based on association data from distinct ancestral populations
  improved IBD prediction for every population in BioMe and significantly improved
  prediction among European ancestry UK Biobank individuals. Lower predictive power
  for non-Europeans was observed, reflecting in part substantially lower African IBD
  case-control reference sizes. We replicated associations for two VEO-IBD genes,
  ADAM17 and LRBA, with high dominant model penetrance in BioMe. Autosomal recessive
  LRBA risk alleles are associated with severe, early-onset autoimmunity; we show
  that heterozygous carriage of an African-predominant LRBA protein-altering allele
  is associated with significantly decreased LRBA and CTLA-4 expression with T-cell
  activation. CONCLUSIONS: Greater genetic diversity in African populations improves
  prediction across populations, and generalizes some VEO-IBD genes. Increasing African
  American IBD case-collections should be prioritized to reduce health disparities
  and enhance pathophysiological insight.'
tags:
- Humans
- Colitis
- Ulcerative
- IBD
- Crohn Disease
- Phenotype
- Genetic Predisposition to Disease
- Penetrance
- United States
- Genome-Wide Association Study
- Polymorphism
- Single Nucleotide
- Risk Factors
- Case-Control Studies
- Jews
- Prevalence
- Risk Assessment
- Age of Onset
- Europe
- Hispanic or Latino
- Multifactorial Inheritance
- Network Meta-Analysis
- PRS
- VEO-IBD
- Race Factors
- Black or African American
- White People
---
